tools/start_workspace_tmux.sh

```diff
@@ -97,7 +97,7 @@ pl_tmux_start_workspace_pane () {
   echo "Starting workspace host..."
   # A background process spams this pane, so pipe it to cat
   # to prevent a shell prompt from appearing also:
-  make start-workspace-host | cat
+  make dev-workspace-host | cat
```

Comment:
@echuber2 what're the desired semantics of this script? That is, should it start PL+workspaces in a way that's suitable for local development of PrairieLearn itself, or for a course? I'm treating it as the former, hence the usage of `dev` scripts here, but I'd like your opionion.

If this script is no longer useful to you, I'd be happy to resolve this question by just deleting it 😅